src/backend/updates.rs

```diff
@@ -70,7 +70,19 @@ fn post_upgrade() {
     });
 }
 
-async fn post_upgrade_fixtures() {}
+async fn post_upgrade_fixtures() {
+    mutate(|state| {
```

Comment:
Shouldn't this be part of `post_upgrade` such that if it fails, then the entire upgrade fails?

Reply:
From our previous discussions about instruction limits I understood that if it's not an actual canister call, the execution happens within the same context. Did I misunderstand this? Because that's indeed what I assume here: that `post_upgrade_fixtures` is "inlined" into `post_upgrade`.

Reply:
But it is subject to the same instruction budged as the parent call?

Reply:
The instruction budget is different as well: upgrade => 200B (for pre/post upgrade combined), timer => 20B.